tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.md

```diff
@@ -181,7 +181,20 @@ This test requires that a vSphere server is running and available
 8. Docker container created with network attached with net1, got ip address inside of network range
 9. Docker create another container, and link to previous one, can talk to the the first container successfully
 
-
+## Container Firewalls
+1. Create an open container and verify another open container can connect to it on arbitrary ports.
+2. Create a closed container and verify an open container cannot connect to it on an arbitrary port.
+3. a. Create an outbound container. Verify first that a container on the bridge cannot connect to the 
```

Comment:
Adding:
* verify that a container on the bridge network *can* connect to the outbound container via its bridge network address when both are connected to the same bridge network - i.e. bridge networks always have "Peer+Published" trust.
* Verify that an outbound container on the same external network cannot connect to the first outbound container.